<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XConfigDialogTX300</class>
 <widget class="QDialog" name="XConfigDialogTX300">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>957</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>957</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>X Configuration Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/tetserver-browser.ico</normaloff>:/tetserver-browser.ico</iconset>
  </property>
  <widget class="QLabel" name="mainPicture">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="XConf.qrc">:/XConf/X Main TX300.bmp</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="offsetPicture">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="XConf.qrc">:/XConf/X sideoff TX300.bmp</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="rotationPicture">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="XConf.qrc">:/XConf/X pitch TX300.bmp</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="planeAngleSpinBox">
      <property name="toolTip">
       <string>Enter the angle between the vertical plane and the active display. If the display is tilted forward you can enter a negative number.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-90.000000000000000</double>
      </property>
      <property name="maximum">
       <double>90.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="planeAngleLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>107</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="planeDistSpinBox">
      <property name="toolTip">
       <string>Enter the distance from the reference point of the Eyetracker to the front of active display. If your Eyetracker is located behind the display, enter a negative distance.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="planeDistLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="planeHeightSpinBox">
      <property name="toolTip">
       <string>Enter the distance from the bottom of the active display (if using a projector setup measure from the bottom of shown image) to the horizontal plane on which the Eyetracker foot stands.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="planeHeightLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>310</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="1">
     <widget class="QLabel" name="pitchLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="pitchSpinBox">
      <property name="toolTip">
       <string>Enter the angle between the back of the Eyetracker and the active display as seen from above. If your eyetracker is rotated clock-wise (i.e. opposite as shown in the picture) enter a negative angle.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="elevationSpinBox">
      <property name="toolTip">
       <string>Enter the angle between the horizontal plane and the body of the eye tracker.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="elevationLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Units:</string>
   </property>
   <widget class="QRadioButton" name="cmRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>[cm]</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="inchRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>[inch]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Active Display Size:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="planeDimWidthSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter the width of the active display, if using a projector setup measure the size of the shown image.</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="planeDimHeightSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&quot;Enter the height of the active display. If using a projector setup measure the size of the shown image.</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>500</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to Eye Tracker</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useSideOffsetCheckBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Side Offset</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="usePitchCheckBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Rotation</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="1">
     <widget class="QLabel" name="sideOffsetLabel">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="sideOffsetSpinBox">
      <property name="toolTip">
       <string>Enter the side offset from the center of the active display to the center of the Eyetracker, enter a negative offset if the Eyetracker is located to the left of the center as seen from the front.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="useTModeCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>621</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Eye Tracker in T Mode (Automatic Configuration).</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFromFileButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load from File...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveToFileButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to File...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="planeHeightSpinBox_2">
      <property name="toolTip">
       <string>Enter the distance from the bottom of the active display (if using a projector setup measure from the bottom of shown image) to the horizontal plane on which the Eyetracker foot stands.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="planeHeightLabel_2">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="elevationSpinBox_2">
      <property name="toolTip">
       <string>Enter the angle between the horizontal plane and the body of the eye tracker.</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>180.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="elevationLabel_2">
      <property name="text">
       <string>[deg]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
  <include location="XConf.qrc"/>
 </resources>
 <connections/>
</ui>
